### ISSUE TITLE
gulp.buble.js - "Failed to load external module buble/register"

### ISSUE THREAD

**Issue by WolfieZero** [js-cli/js-interpret#42]
I'm trying to use ES2015 in my Gulpfile with buble, but I get the error `Failed to load external module buble/register`

package.json

```
"devDependencies": {
    ...
    "buble": "^0.13.2",
    "gulp": "^3.9.1",
    ...
}
```

**Comment by kaisermann** [comment]
Having the same issue here.

**Comment by phated** [comment]
Fixed in v1.0.3
